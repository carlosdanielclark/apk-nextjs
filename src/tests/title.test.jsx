import { mount } from 'enzyme';
import FormTitle from '../app/title';

describe('Tests in "FormTitle"', () => {


    test('Snapshot of the html', () => {
        //(1) initialization
        const wrapper = mount( <FormTitle /> );
        //(2) Stimulus  and  //(3) Observation 
        expect( wrapper ).toMatchSnapshot();
    });

    test('Show title correctly', () => {
        //(1) initialization
        const wrapper = mount( <FormTitle /> );
        //(2) Stimulus 
        const textParagraph = wrapper.find('label').text().trim();
        //(3) Observation 
        expect(textParagraph).toBe('Currency Exchange!');
    });
  

})