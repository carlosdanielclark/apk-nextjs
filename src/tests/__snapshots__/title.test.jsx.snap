// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tests in "Form" > Must show correctly 1`] = `
<div
  className="_Title_63b601"
>
  <label>
    Currency Exchange!
  </label>
</div>
`;

exports[`Tests in "FormTitle" > Must show correctly 1`] = `
<div
  className="_Title_63b601"
>
  <label>
    Currency Exchange!
  </label>
</div>
`;

exports[`Tests in "FormTitle" > Snapshot of the html 1`] = `
<div
  className="_Title_63b601"
>
  <label>
    Currency Exchange!
  </label>
</div>
`;
